tests/E2E/EntryBasicTest.php

```diff
@@ -24,20 +25,43 @@ public function setUp()
     }
 
     /**
-     * @vcr e2e_entry_get_all.json
+     * @vcr e2e_entry_get_all_locale_all.json
      */
     public function testGetAll()
     {
+        $query = (new Query())
+            ->setLocale('*');
+        $assets = $this->client->getEntries($query);
+
+        $this->assertInstanceOf(ResourceArray::class, $assets);
+    }
+
+    /**
+     * @vcr e2e_entry_get_all_locale_default.json
+     */
+    public function testGetAllDefaultLocale()
+    {
         $assets = $this->client->getEntries();
 
```

Comment:
The main point in these tests is that it doesn't fall over and throws an exception when constructing the classes. Though more assertions wouldn't hurt ;)